client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -1,7 +1,11 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
```

Comment:
did you use useState or useEffect?

Reply:
I plan on trying these hooks out. I has some code written using them, but deleted that before pushing this commit. I plan on going back and using them.